core/arch/arm32/plat-vexpress/conf.mk

```diff
@@ -64,6 +64,10 @@ include mk/config.mk
 
 CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= 1
 
+ifeq ($(CFG_ENC_FS),y)
+CFG_SST_KEY_MANAGER_TEST := y
```

Comment:
We're using enc_fs or sst for the same functionality. Can we unify the names (defines / files / functions)